code/obj/window.dm

```diff
@@ -836,6 +845,26 @@ ADMIN_INTERACT_PROCS(/obj/window, proc/smash)
 		for (var/obj/grille/G in orange(1,src))
 			G.UpdateIcon()
 
+	proc/update_damage_overlay()
+		var/health_percentage = health/health_max
+		if (!src.damage_image)
+			src.damage_image = image('icons/obj/window_damage.dmi')
+			src.damage_image.appearance_flags = RESET_COLOR | RESET_ALPHA
```

Comment:
This makes it so when scaled / transformed this overlay doesn't get bilinearly interpolated which looks blurry and doesn't fit with pixel art.
```suggestion
			src.damage_image.appearance_flags = PIXEL_SCALE | RESET_COLOR | RESET_ALPHA
```